includes/elements/container.php

```diff
@@ -296,18 +311,18 @@ protected function register_container_layout_controls() {
 			[
 				'label' => esc_html__( 'Content Width', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
-				'default' => 'boxed',
+				'default' => 'full',
 				'options' => [
 					'boxed' => esc_html__( 'Boxed', 'elementor' ),
 					'full' => esc_html__( 'Full Width', 'elementor' ),
 				],
-				'render_type' => 'ui',
+				'render_type' => 'template',
 				'selectors' => [
 					'{{WRAPPER}}' => '{{VALUE}}',
 				],
 				'selectors_dictionary' => [
-					'boxed' => '',
-					'full' => '--content-width: 100%;',
+					'boxed' => '--container-flex-direction: column; --container-flex-grow: 1;',
```

Comment:
Why change the flex direction?

Reply:
@hein-obox

Reply:
@udidol 
In case we of the boxed width I want to avoid that we would apply all flex styling both to the container as the inner box. 
I have set most of the initial values for the flex container and have applied the controls to the inner div. 

All flex item styling I have applied to the container and set default values for the inner div. 

e.g. 
![image](https://user-images.githubusercontent.com/94352322/183911324-4a781115-1897-4679-85c2-0afd5c478f58.png)

![image](https://user-images.githubusercontent.com/94352322/183911203-6bdcd4cb-4397-411e-9490-16cd4937b4a3.png)